src/db_connect/models/book.js

```diff
@@ -0,0 +1,29 @@
+const { Sequelize, DataTypes } = require("sequelize");
+
+module.exports = (sequelize) => {
+  const Book = sequelize.define("books", {
+    isbn: {
+      type: DataTypes.INTEGER,
```

Comment:
Changed ISBN to text field. As discussed, leaving book_id field for a later version. This will probably cause some backwards-compatibility/update problems. Oh well.